### PR TITLE
ign -> gz Namespace Migration : gz-fuel-tools

### PR DESCRIPTION
**Pull request by methylDragon**
See https://github.com/gazebo-tooling/release-tools/issues/711

### CODE REVIEW

File: Migration.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-## Ignition Fuel Tools 5.X to 6.X
+## Ignition Fuel Tools 7.X to 8.X
```
Comment (chapulina) [inline Migration.md:1]: We should add a new section, I think the previous content actually belonged to 5->6

File: include/gz/fuel_tools/config.hh.in [hunk 2]
```diff
@@ -1,17 +1,39 @@
...
+#define GZ_FUEL_TOOLS_MINOR_VERSION ${PROJECT_VERSION_MINOR}
+#define GZ_FUEL_TOOLS_PATCH_VERSION ${PROJECT_VERSION_PATCH}
 
-#define IGNITION_FUEL_TOOLS_VERSION "${PROJECT_VERSION}"
-#define IGNITION_FUEL_TOOLS_VERSION_FULL "${PROJECT_VERSION_FULL}"
+#define GZ_FUEL_TOOLS_VERSION "${PROJECT_VERSION}"
+#define GZ_FUEL_TOOLS_VERSION_FULL "${PROJECT_VERSION_FULL}"
 
-#define IGNITION_FUEL_TOOLS_VERSION_HEADER "${PROJECT_NAME_LOWER}, version ${PROJECT_VERSION_FULL}\nCopyright (C) 2017 Open Source Robotics Foundation.\nReleased under the Apache 2.0 License.\n\n"
+#define GZ_FUEL_TOOLS_VERSION_HEADER "Gz Fuel Tools, version ${PROJECT_VERSION_FULL}\nCopyright (C) 2017 Open Source Robotics Foundation.\nReleased under the Apache 2.0 License.\n\n"
```
Comment (chapulina) [inline include/gz/fuel_tools/config.hh.in:31]:
```suggestion
#define GZ_FUEL_TOOLS_VERSION_HEADER "Gazebo Fuel Tools, version ${PROJECT_VERSION_FULL}\nCopyright (C) 2017 Open Source Robotics Foundation.\nReleased under the Apache 2.0 License.\n\n"
```

**Review by chapulina — APPROVED** [review]
LGTM with 🟢 CI

File: src/WorldIter_TEST.cc [hunk 3]
```diff
@@ -44,8 +44,8 @@ namespace ignition
 
       public: static WorldIter WorldIterThreeWorldIds()
         {
-          ignition::fuel_tools::ServerConfig srv;
-          srv.SetUrl(ignition::common::URI("https://ignitionrobotics.org"));
+          gz::fuel_tools::ServerConfig srv;
+          srv.SetUrl(gz::common::URI("https://gazebosim.org"));
```
[the remark this answers is not in the document]
Comment (methylDragon) [reply src/WorldIter_TEST.cc]: Gotcha, let me revert.
I didn't see changes for github URLs, so I left those in. Let me know if I should remove those as well!

https://github.com/gazebosim/gz-fuel-tools/pull/251/commits/701189cf832f227222b7a1aeb8ebe6eb0ddebbaf

[the remark this answers is not in the document]
Comment (chapulina) [reply src/WorldIter_TEST.cc]: The GitHub URLs are good, thanks!

**Review by chapulina — APPROVED** [review]
LGTM with 🟢 CI

**Pull request merged**